pyramid/tier_fs_test.go

```diff
@@ -17,7 +23,7 @@ import (
 
 var (
 	fs      FS
-	adapter block.Adapter
+	adapter *memAdapter
```

Comment:
Can we just avoid global state and create its test with its own adapter?  As is we cannot run tests in parallel, and we need weird checks on the `gets` counter.

Reply:
Yes - fixed.